flaml/online_automl/auto_vw.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+from typing import Optional
+import logging
+from ..tune.trial import Trial
+from ..tune.online_trial_runner import OnlineTrialRunner
+from ..scheduler.online_scheduler import ChaChaScheduler
+from ..searcher.online_searcher import ChampionFrontierSearcher
+logger = logging.getLogger(__name__)
+
+
+class AutoVW:
+    """Implemention of AutoVW
+
+    Args:
+        ************Args about the task***************
+        problem: problem, which is instance of VWNameSpaceInteraction. It provides task related info: 
```

Comment:
This argument is not very intuitive from the user's perspective. Same for VWNameSpaceInteraction, trainable_func. init_config and search_space are easy to understand, but examples should be provided. Please check the docstr for tune.run() or AutoML.fit() and try to make the API easy to use and understand.

Reply:
Actually, this docstr should not be placed here.

Reply:
We probably should remove the concept of 'problem' in the API. We can keep init_config and search_space. Not sure about trainable_func. Might need a different name.